src/main/java/com/autilus/hw2806/MailMessage.java

```diff
@@ -0,0 +1,7 @@
+package com.autilus.hw2806;
+
+public static class MailMessage extends Send<String> {
```

Comment:
do we need static here?

Reply:
I think no, but in the conditions of the task, static

Reply:
in the conditions of the task we use static because all of this classes are nested. But in your implementation you create all classes at the separate files. So we can remove 'static'.

Reply:
ok), I understand